printer_mods/jasonc/Z_Idler_Shoulder_Bolt/README.md

```diff
@@ -0,0 +1,13 @@
+# Voron v2.4 Z Idler shoulder bolt modification
+
+Nothing too special, just some small tweaks to the OEM shape to allow for the shoulder bolt. There's a cross-section drawing of the modifications I did, and I used the McMaster-Carr part number: 90265A113 - a 5x20mm shoulder bolt (m4 threaded). You can also purchase them through [Filastruder](https://www.filastruder.com/products/shoulder-screws-bolts?_pos=7&_sid=b4b266eda&_ss=r), they are the cheapest source.
```

Comment:
I looked up that McMaster-Carr part number and it appears to be a 3x6mm shoulder bolt with M2 threads. Perhaps you meant [90265A133](https://www.mcmaster.com/90265A133)?

Reply:
You are correct, it's a typo - fixed.